src/qcodes/instrument_drivers/cryomagnetics/Cryomagnetics4G_visa.py

```diff
@@ -0,0 +1,287 @@
+import re
+import time
+from dataclasses import dataclass
```

Comment:
The publicly exposed qcodes drivers are exposed at the level qcodes.instrument_drivers.vendor so please import everything that you want to be public in the `__init__.py` file and add to `__all__` too. (you can compare with AMI drivers for example)

Reply:
Thanks! Yes that's good to remeber :)